shared/src/consensus/mod.rs

```diff
@@ -13,6 +13,16 @@ pub struct ConsensusFault {
     pub fault_type: ConsensusFaultType,
 }
 
+/// Result of checking two headers for a consensus fault, with the gas used
+/// Only for v14 and earlier
+#[derive(Clone, Debug)]
+pub struct ConsensusFaultWithGas {
+    /// The fault.
+    pub fault: Option<ConsensusFault>,
+    /// Gas used in checking the fault
+    pub gas_used: i64,
+}
```

Comment:
1. There's no real need for a named type here. We can just use `(Option<ConsensusFault>, i64)`.
2. This doesn't really belong in "shared" (shared is for types shared between actors and the FVM, not the FVM and lotus).